lib/pbench/test/unit/server/query_apis/test_table_of_content.py

```diff
@@ -0,0 +1,206 @@
+from http import HTTPStatus
+
+import pytest
+
+from pbench.server.api.resources.query_apis.datasets.table_of_content import (
+    TableOfContent,
+)
+from pbench.server.database.models.datasets import Dataset
+from pbench.test.unit.server.query_apis.commons import Commons
+
+
+class TestTableOfContents(Commons):
+    """
+    Unit testing for TableOfContent class.
```

Comment:
Does anyone else think that "table of contents" should be plural? (@dbutenhof, @npalaska, @bueller?)

If not, then, Riya, you should probably un-plural-ize it in line 12 here....  😢

Reply:
I agree, and I already commented on that elsewhere albeit in a slightly different context.

It's also interesting that the test class name _is_ plural, while referring to the singular tense class name it's testing, so that's an additional inconsistency.

Reply:
haha.. sure :)